PokemonApp/Controller/ViewController.swift

```diff
@@ -275,9 +273,24 @@ extension ViewController: UISearchBarDelegate {
                 if pokemon.pokemonName.contains(searchText.lowercased()) {
                     pokemonesFiltrados?.append(pokemon)
                 }
-                boolVar = false
+                activeButtonOne = false
             }
         }
         self.tableView.reloadData()
     }
 }
+
+extension ViewController: DatosPokemon {
+    func datosPokemon(name: String, tipo: String, stats: [Float]?, codigoPokemon: String) {
+    }
+    
+    func updateDelegate() {
```

Comment:
this name is wrong, please use a name that is more according to the purpose of the functionality